features/snippet.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Snippet life cycle
+
+    Scenario: Creating a snippet
+        Given I am authenticated
+        When I fill in the snippet creation form
```

Comment:
You are absolutely right: it makes much more sense to specify the page where the action has to take place !
Concerning the indentation, I didn't notice that I had 3 spaces in my feature files... I will have to check how to change vim settings to play well with cucumber.